iam_openapi_spec.yml

```diff
@@ -1982,6 +2013,7 @@ components:
       additionalProperties: false
       required:
         - name
+        - passcode
```

Comment:
Lets not make passcode as required parameter for creating an organization. Create organization will be authenticated either when
1. When client pass the passcode
2. Or the client pass the secret key in the "x-api-key" header.

This will help in cases where our support team creates organization for a business.